src/pages/manager/BankManagerMainPage.js

```
@@ -3,9 +3,29 @@ import { expect } from '@playwright/test';
 export class BankManagerMainPage {
   constructor(page) {
     this.page = page;
+    this.addCustomerLink = page.locator('button').filter({hasText:'Add Customer'})
+    this.openAccountLink = page.locator('button').filter({hasText:'Open Account'})
+    this.customersListLink = page.locator('button').filter({hasText:'Customers'})
   }
 
-  async open() {
+  async openManagerMainPage() {
     await this.page.goto('/angularJs-protractor/BankingProject/#/manager');
   }
-}
+
+   async waitManagerMainPage() {
+    await this.page.waitForURL('/angularJs-protractor/BankingProject/#/manager');
+  }
+
+  async clickCustomerListLink () {
+    await this.customersListLink.click()
+  }
+
+  async clickAccountLink () {
+    await this.openAccountLink.click()
+  }
+
+  async elementVisible (element) {
+    await expect(element).toBeVisible()
```

Method naming: `elementVisible(element)` performs an assertion using Playwright's `expect`. Consider renaming to `expectElementVisible` or `assertElementVisible` to make it clear that this method asserts (throws on failure) rather than returning a boolean. This is a readability/intent suggestion to avoid confusion when reading tests.